packages/api/src/external/commonwell/patient.ts

```diff
@@ -406,6 +394,30 @@ export async function update(
   }
 }
 
+async function queryDocsIfScheduled(
+  patient: Patient,
+  getOrgIdExcludeList: () => Promise<string[]>
+): Promise<void> {
+  const updatedPatient = await getPatientOrFail(patient);
```

Comment:
Discussed offline, it makes sense to query again since we do a few things that take a while between the create/update are called and this is executed, which increases considerably the probability of having a DQ in between.